src/components/molecules/InternalChevronAccordion/InternalChevronAccordion.tsx

```diff
@@ -88,6 +109,36 @@ const Accordion = ({
   id,
   ...styleProps
 }: InternalChevronAccordionProps) => {
+  const [shouldDisplayShadow, setShouldDisplayShadow] = useState(false);
+  const scrollBox = useRef(null as null | HTMLDivElement);
```

Comment:
interesting why does null need the type assertion?

Reply:
I was getting type errors but actually it should be this:
`const scrollBox = useRef<HTMLDivElement>(null);`

updated